<ui version="4.0" >
 <class>BuscarClienteDialogClass</class>
 <widget class="QDialog" name="BuscarClienteDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Buscar Cliente</string>
  </property>
  <property name="windowIcon" >
   <iconset>images/BCliente.PNG</iconset>
  </property>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Codigo</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Direccion</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="lineEdit_3" />
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="lineEdit_2" />
    </item>
    <item row="2" column="1" >
     <widget class="QLineEdit" name="lineEdit" />
    </item>
    <item row="3" column="1" >
     <widget class="QLineEdit" name="lineEdit_4" />
    </item>
    <item row="4" column="1" >
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton" >
        <property name="text" >
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2" >
        <property name="text" >
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Tel√©fono</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>BuscarClienteDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel" >
     <x>192</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>BuscarClienteDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>332</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel" >
     <x>315</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
